ports/vcpkg-cmake/vcpkg_cmake_configure.cmake

```
@@ -100,13 +100,21 @@ function(vcpkg_cmake_configure)
             "${VCPKG_CMAKE_SYSTEM_NAME}-${VCPKG_TARGET_ARCHITECTURE}-${VCPKG_PLATFORM_TOOLSET}")
     endif()
 
+    set(parallel_log_args "")
+    set(log_args "")
+
     if(generator STREQUAL "Ninja")
         vcpkg_find_acquire_program(NINJA)
         vcpkg_list(APPEND arg_OPTIONS "-DCMAKE_MAKE_PROGRAM=${NINJA}")
         # If we use Ninja, it must be on PATH for CMake's ExternalProject,
         # cf. https://gitlab.kitware.com/cmake/cmake/-/issues/23355.
         get_filename_component(ninja_path "${NINJA}" DIRECTORY)
         vcpkg_add_to_path("${ninja_path}")
+        set(parallel_log_args
+            "../build.ninja" ALIAS "rel-ninaja.log"
+            "../../${TARGET_TRIPLET}-dbg/build.ninja" ALIAS "dbg-ninaja.log"
```

```suggestion
            "../build.ninja" ALIAS "rel-ninja.log"
            "../../${TARGET_TRIPLET}-dbg/build.ninja" ALIAS "dbg-ninja.log"
```
but I'm not sure if they are so useful.

---

Useful for e.g. debugging link issues where the install logs just shows the response file being used.

---

I understand the problem with the response files, but does the ninja file at more information? (Dependencies, of course, but content?)